server/models/adapter.go

```diff
@@ -8,18 +8,19 @@ import (
 
 // Available Meshery adapters
 var (
-	Istio   = Adapter{Name: "meshery-istio", Location: "10000"}
-	Linkerd = Adapter{Name: "meshery-linkerd", Location: "10001"}
-	Consul  = Adapter{Name: "meshery-consul", Location: "10002"}
-	NSM     = Adapter{Name: "meshery-nsm", Location: "10004"}
-	AWS     = Adapter{Name: "meshery-app-mesh", Location: "10005"}
-	Traefik = Adapter{Name: "meshery-traefik-mesh", Location: "10006"}
-	Kuma    = Adapter{Name: "meshery-kuma", Location: "10007"}
-	Nginx   = Adapter{Name: "meshery-nginx-sm", Location: "10010"}
-	Cilium  = Adapter{Name: "meshery-cilium", Location: "10012"}
+	Istio     = Adapter{Name: "meshery-istio", Location: "10000"}
+	Linkerd   = Adapter{Name: "meshery-linkerd", Location: "10001"}
+	Consul    = Adapter{Name: "meshery-consul", Location: "10002"}
+	NSM       = Adapter{Name: "meshery-nsm", Location: "10004"}
+	AppMesh   = Adapter{Name: "meshery-app-mesh", Location: "10005"}
+	Traefik   = Adapter{Name: "meshery-traefik-mesh", Location: "10006"}
+	Kuma      = Adapter{Name: "meshery-kuma", Location: "10007"}
+	Nginx     = Adapter{Name: "meshery-nginx-sm", Location: "10010"}
+	Cilium    = Adapter{Name: "meshery-cilium", Location: "10012"}
+	Nighthawk = Adapter{Name: "meshery-nighthawk", Location: "10013"}
 )
 
-var ListAvailableAdapters = []Adapter{Istio, Linkerd, Consul, Kuma, NSM, Nginx, AWS, Cilium, Traefik}
+var ListAvailableAdapters = []Adapter{Istio, Linkerd, Consul, Kuma, NSM, Nginx, AppMesh, Cilium, Traefik}
```

Comment:
`Nighthawk` will be placed here when the meshery-nighthawk adapter is ready for use, right?

Reply:
Good catch. My intention in this PR is to get Nighthawk entries fully ready, but to perhaps, leave it out its container deployment from the default list of containers to deploy. In other words, I missed this update. Thank you for pointing this area out. 

In this or other PR, we'll want to add the sub-chart for the new adapter. I think that you have write permission to this branch, so you are most welcome to make any changes (add commits) to this branch as you see fit, too. You won't be stepping on my toes. 👍

Reply:
I went ahead and added "Nighthawk" to the list. 👍